packages/bodiless-organisms/src/components/Menu/MegaMenu.token.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright © 2020 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flow } from 'lodash';
+import { useEditContext } from '@bodiless/core';
+import { withDesign, addClasses, addClassesIf } from '@bodiless/fclasses';
+
+/**
+ * Utility Styles
+ * ===========================================
+ */
+const isActive = () => useEditContext().isActive;
+const isNotActive = () => !useEditContext().isActive;
+
+const asVerticalSubMenu = withDesign({
+  Wrapper: withDesign({
+    List: addClasses('flex-col'),
+  }),
+});
+
+const asStaticOnHover = withDesign({
+  Wrapper: withDesign({
+    WrapperItem: addClasses('hover:static'),
+  }),
+});
+
+const asRelative = withDesign({
+  Wrapper: withDesign({
+    WrapperItem: addClasses('relative'),
+  }),
+});
+
+const asRelativeNotActive = withDesign({
+  Wrapper: withDesign({
+    WrapperItem: addClassesIf(isNotActive)('relative'),
+  }),
+});
+
+const asExpandedOnActive = withDesign({
+  Wrapper: withDesign({
+    WrapperItem: addClassesIf(isActive)('overflow-visible'),
+  }),
+});
+
+const asFullWidthSublist = withDesign({
+  Wrapper: withDesign({
+    List: addClasses('w-full'),
+  }),
+});
+
+const asResponsiveSublist = withDesign({
+  Wrapper: withDesign({
+    List: addClasses('w-content min-w-full'),
+  }),
+});
+
+/**
+ * Base Menu Styles
+ * ===========================================
+ */
+const withBaseMenuStyles = withDesign({
+  Wrapper: addClasses('relative flex'),
+  Item: addClasses('overflow-hidden hover:overflow-visible'),
+});
+
+/**
+ * Base Sub Menu Styles
+ * ===========================================
+ */
+const withBaseSubMenuStyles = withDesign({
+  Wrapper: withDesign({
+    List: addClasses('flex absolute left-0'),
+  }),
+});
+
+/**
```

Comment:
Again, split the simple menu tokens into a separate file, and include any shared tokens in that file.

Reply:
Done